auth/user_mgt.go

```diff
@@ -416,6 +427,14 @@ func validatePhone(val interface{}) error {
 	return nil
 }
 
+func validateValidSince(val interface{}) error {
+	if v, ok := val.(int64); !ok || v > 1e12 {
```

Comment:
Is this to validate the UpdateUser() call? If so it should accept a milliseconds timestamp and convert it into seconds.

Reply:
we are setting the validSince property directly. (from a private function)
validSince is in seconds, TokensValidAfterTime is in millis.

Reply:
Then please remove this check.